subsys/logging/Kconfig

```diff
@@ -279,6 +279,40 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FLASH
```

Comment:
I wonder if this backend should not be called `LOG_BACKEND_FS`  (or `LOG_BACKEND_FILESYSTEM`) since it is using file system API rather than flash API.

Reply:
Replaced by `LOG_BACKEND_FS` prefix.